plutus-example/plutus-example/src/Cardano/PlutusExample/ScriptContextChecker.hs

```diff
@@ -327,20 +333,15 @@ readCustomRedeemerFromTx fp (AnyConsensusModeParams cModeParams) network = do
                    (ConsensusModeMismatch (AnyConsensusMode CardanoMode) (AnyCardanoEra cEra))
                    $ toEraInMode cEra CardanoMode
 
-      eResult <-
-        liftIO $ executeLocalStateQueryExpr localNodeConnInfo Nothing
-          $ \ntcVersion -> do
```

Comment:
We no longer need to worry about `ntcVersion` lambda when executing queries because of the use of `ReaderT`.